killrweather-examples/src/main/scala/com/datastax/killrweather/SampleJson.scala

```diff
@@ -61,6 +61,6 @@ object GitHubEvents {
   case class MonthlyCommits(user: String, commits: Int, month: Int, year: Int) extends Commits
   object MonthlyCommits {
     def apply(r: Row): MonthlyCommits = MonthlyCommits(
-      r.getString(0), r.getInt(1), r.getInt(2), r.getInt(3))
+      r.getString(0), r.getLong(1).toInt, r.getLong(2).toInt, r.getLong(3).toInt)
```

Comment:
@bkhadige I am not clear on why you want to change Int types to Long? This is the schema

https://github.com/bkhadige/killrweather/blob/patch-1/killrweather-examples/src/main/scala/com/datastax/killrweather/SampleJson.scala#L36-L40